crates/astria-sequencer/src/transaction/mod.rs

```diff
@@ -269,6 +268,10 @@ impl ActionHandler for SignedTransaction {
                     .await
                     .context("failed executing bridge sudo change")?,
             }
+            state
+                .increment_transaction_index_of_action()
```

Comment:
As above, why is it necessary to write this to state?

Can this just reside in the ephemeral state instead?